main.py

```diff
@@ -48,6 +48,8 @@ def cards(pages, columns=1, style="cards", force_version=False):
         canonical = current_page.canonical_url
         url_parts = re.search("//([^/]+)/([^/]+)/([^/]+)/([^/]+)/([^/]+)/", canonical)
         (site, project, edition, language, version) = url_parts.groups()
```

Comment:
`site` is not really useful. Those two lines should be removed, and `site` removed below from the rebuilt URLs passed to the cards.